assets/airports/klga.json

```diff
@@ -0,0 +1,4810 @@
+                "SHLBK": ["SHLBK", ["LOUIE", "A130"], "BACKY", "ODESA", "DQO", "RUUTH"],
+                "RUUTH": ["RUUTH"]
+            },
+            "body": ["WNDYL", ["BRAND", "A80"], "KORRY", "DEPDY", "TYKES", "MINKS", "DAMYR", "RENUE", "APPLE", "PROUD", "VADDR", "#045"],
+            "rwy": {
+                "KLGA04": [],
+                "KLGA13": [],
+                "KLGA22": [],
+                "KLGA31": []
+            },
+            "draw": [
+                ["SWANN*", "GATBY", "KERNO", "ODESA"],
+                ["SHLBK*", "LOUIE", "BACKY", "ODESA", "DQO", "RUUTH"],
+                ["RUUTH", "WNDYL", "BRAND", "KORRY", "DEPDY", "TYKES", "MINKS", "DAMYR", "RENUE", "APPLE", "PROUD", "VADDR"]
+            ]
+        },
+        "HAARP3": {
+            "icao": "HAARP3",
+            "name": "Harp Three",
+            "entryPoints": {
+                "RKA": ["RKA", "PETER", "CYPER", "TRESA", "IGN", "VALRE"],
+                "ALB": ["ALB", "CTR", "PWL", "IGN", "VALRE"],
+                "VALRE": ["VALRE"]
+            },
+            "body": [["BASYE", "A080"], "HAARP", "CRALY", "LGA"],
+            "rwy": {
+                "KLGA04": [],
+                "KLGA13": [],
+                "KLGA22": [],
+                "KLGA31": []
+            },
+            "draw": [
+                ["RKA*", "PETER", "CYPER", "TRESA", "IGN"],
+                ["ALB*", "CTR", "PWL", "IGN", "VALRE"],
+                ["VALRE", "BASYE", "HAARP", "CRALY", "LGA"]
+            ]
+        },
+        "KORRY4": {
+            "icao": "KORRY4",
+            "name": "Kory Four",
+            "entryPoints": {
+                "AGARD": ["AGARD", "SPEAK"],
+                "GVE": ["GVE", "COLIN", "PXT", "GARED", ["RIDGY", "A270+"]],
+                "PXT": ["PXT", "GARED", ["RIDGY", "A270+"]],
+                "ENO": []
+            },
+            "body": [["ENO", "A240+"], "SKIPY", ["BESSI", "A170+"], "EDJER", ["DAVYS", "A130+"], "HOLEY", ["KORRY", "A100"], "RBV", "TYKES", "MINKS", "RENUE", "APPLE", "PROUD", "LGA"],
+            "rwy": {
+                "KLGA04": [],
+                "KLGA13": [],
+                "KLGA22": [],
+                "KLGA31": []
+            },
+            "draw": [
+                ["AGARD", "SPEAK", "ENO"],
+                ["GVE*", "COLIN", "PXT"],
+                ["PXT", "GARED", "RIDGY", "ENO"],
+                ["ENO", "SKIPY", "BESSI", "EDJER", "DAVYS", "HOLEY", "KORRY", "RBV", "TYKES", "MINKS", "RENUE", "APPLE", "PROUD", "LGA"]
+            ]
+        },
+        "MIP4": {
+            "icao": "MIP4",
+            "name": "Milton Four",
+            "entryPoints": {
+                "ETG": ["ETG"],
+                "PSB": ["PSB"],
+                "SFK": ["SFK"],
+                "MIP": []
+            },
+            "body": ["MIP", "MARRC", ["BILEY", "A130"], "VIBES", "FJC", "LIZZI", ["BEUTY", "A100"], "HARLM", "DREMS", "APPLE", "PROUD", "LGA"],
```

Comment:
MARRC should be A180

Reply:
@Fechulo Regarding MARRC here, and PETER below, I thought that "Expect" altitudes were not supposed to be encoded in the STAR. Is that no longer the case?

Reply:
Generally I would agree, but since you included the restrictions at BILEY and BEUTY, it makes sense to include them all